tests/rotatingMHD/time_discretization.output

```diff
@@ -21,6 +21,69 @@ Step =      0, Current time = 0.000000e+00, Next time step = 1.000000e-01
     |    gamma    |   1.00e+00 |   0.00e+00 |   0.00e+00 | 
     |  extra_pol  |     -      |   1.00e+00 |   0.00e+00 | 
     +-------------+------------+------------+------------+
+Step =      1, Current time = 1.00e-01, Next time step = 1.00e-01
+    +-------------+------------+------------+------------+
+    |    Index    |     n      |    n-1     |     n-2    |
+    +-------------+------------+------------+------------+
+    |    alpha    |   1.50e+00 |  -2.00e+00 |   5.00e-01 | 
+    |    beta     |     -      |   2.00e+00 |  -1.00e+00 | 
+    |    gamma    |   1.00e+00 |   0.00e+00 |   0.00e+00 | 
+    |  extra_pol  |     -      |   2.00e+00 |  -1.00e+00 | 
+    +-------------+------------+------------+------------+
+Step =      2, Current time = 2.00e-01, Next time step = 1.00e-01
+Step =      3, Current time = 3.00e-01, Next time step = 1.00e-01
+Step =      4, Current time = 4.00e-01, Next time step = 1.00e-01
+Step =      5, Current time = 5.00e-01, Next time step = 1.00e-01
+Step =      6, Current time = 6.00e-01, Next time step = 1.00e-01
+Step =      7, Current time = 7.00e-01, Next time step = 1.00e-01
+Step =      8, Current time = 8.00e-01, Next time step = 1.00e-01
+Step =      9, Current time = 9.00e-01, Next time step = 1.00e-01
+Step =     10, Current time = 1.00e+00, Next time step = 0.00e+00
+Step =     10, Current time = 1.00e+00, Next time step = 2.00e-01
+    +-------------+------------+------------+------------+
+    |    Index    |     n      |    n-1     |     n-2    |
+    +-------------+------------+------------+------------+
+    |    alpha    |   1.67e+00 |  -3.00e+00 |   1.33e+00 | 
+    |    beta     |     -      |   3.00e+00 |  -2.00e+00 | 
+    |    gamma    |   1.00e+00 |   0.00e+00 |   0.00e+00 | 
+    |  extra_pol  |     -      |   3.00e+00 |  -2.00e+00 | 
+    +-------------+------------+------------+------------+
```

Comment:
The changed values are due to resuming with a new time step, correct?

Reply:
I will check this.

Reply:
I will check this...